### ISSUE TITLE
$jobs->create not working for consumers

### ISSUE THREAD

**Issue by semichkin** [spiral/roadrunner-jobs#6]
i can't read tasks from `$jobs->create` created queue

worker code:
```php
<?php

include realpath(__DIR__ . "/../bootstrap.php");

use Spiral\Goridge\RPC\RPC;
use Spiral\Goridge\StreamRelay;
use Spiral\RoadRunner;
use Spiral\RoadRunner\Environment;

ini_set('display_errors', 0);
error_reporting(0);

$env = Environment::fromGlobals();
$rpc = RPC::create($env->getRPCAddress());

$jobs = new RoadRunner\Jobs\Jobs($rpc);

$p = $jobs->create(new RoadRunner\Jobs\Queue\AMQPCreateInfo(
    'myqueue',
    1,
    1,
    'myqueue',
    'amqp.default1',
));
$p->resume();

$consumer = new RoadRunner\Jobs\Consumer();

while ($task = $consumer->waitTask()) {
    echo PHP_EOL . "taskk";

    $task->complete();
}
```

rr.yaml config:

```yaml
rpc:
  listen: tcp://127.0.0.1:6001

server:
  command: "php workers/bot_queue_worker.php"
  relay: pipes

logs:
  level: debug

amqp:
  addr: amqp://guest:guest@rabbitmq:5672

jobs:
  consume: ["pipe1", "pipe2"]
  num_pollers: 16
  pipeline_size: 100000
  timeout: 1
  pool:
    num_workers: 10
    max_jobs: 0
    allocate_timeout: 60s
    destroy_timeout: 60s
```

**Comment by butschster** [comment]
Did you configured pipelines in `rr.yaml`?

```yaml
amqp:
  addr: amqp://guest:guest@localhost:5672

jobs:
  pipelines:
    pipe1:
      driver: amqp
    pipe2:
      driver: amqp
```

I have working example of queue worker in my [project](https://github.com/buggregator/app/tree/feature/roadrunner/app/Application/Queue). Feel free to ask questins.

**Comment by semichkin** [comment]
@butschster no, $jobs->create uses for describe pipes in code dynamically, not in configs

**Comment by butschster** [comment]
@semichkin Thx you for your contribution!

**Comment by rustatian** [comment]
@semichkin Will be fixed in the RR `v2.6.0`.

**Comment by rustatian** [comment]
The issue was fixed in `2.6.0`. @semichkin Feel free to re-open it if you see it again.
